packages/graphback-codegen-schema/tests/snapshots/GraphQLSchemaCreatorTest.ts.md

```diff
@@ -123,6 +125,7 @@ Generated by [AVA](https://ava.li).
       createTest(input: TestInput): Test!␊
       updateTest(input: TestInput): Test!␊
       deleteTest(input: TestInput): Test!␊
+      likeNote(id: ID!): Note!␊
```

Comment:
It is sometimes as easy as checking the outputs.
I rarely did that before by simply skipping those files but they are actually the most important ones to review first.